src/styles/mixins/_focus-style.scss

```diff
@@ -0,0 +1,24 @@
+@mixin sb-focus-style($focus-radius: 4px) {
+  &:focus-visible,
+  &.focus-visible {
+    @include sb-focus-style-css($focus-radius);
+  }
+
+  &:focus:not(.focus-visible) {
+    outline: none;
+  }
+}
+
+@mixin sb-focus-style-css($focus-radius: 4px, $shadow-depth: 3px) {
+  @include sb-focus-style-base($focus-radius);
+
+  box-shadow:
+    0 0 0 $shadow-depth hsl(209deg 100% 50% / 50%),
+    0 0 0 1px var(--sb-primary-hover-color) inset;
+}
+
+@mixin sb-focus-style-base($focus-radius: 4px) {
```

Comment:
is it a copuy paste? the z-index 11 is weird... what is the purpose of this sub mixin?

Reply:
Copy paste, don't really know why it's like that